src/App.jsx

```diff
@@ -1,15 +1,9 @@
 import React from 'react';
 
-import { Counter, Layout } from './components';
+import { Home } from './Views/Home/Home';
```

Comment:
Should we write `views` with lowercase for correct import?
`import { Home } from './views/Home/Home';`